### PR TITLE
Added logic to hide/show chip number field

### PR DESCRIPTION
**Pull request by gary-wood**

### CODE REVIEW

File: app/src/js/main.js [hunk 1]
```diff
@@ -4,11 +4,18 @@ petDetectrApp.formSetup = function formSetup() {
...
     // add event listeners to radio buttons
     for (let i = 0, max = this.petStatus.length; i < max; i++) {
         this.petStatus[i].addEventListener('click', this.updateDateAndLocationText, true);
     }
+    
+    // event listener for chip number toggle
+    for (let i = 0; i < this.isChippedSelector.length; i++) {
+        this.isChippedSelector[i].addEventListener('click', this.hideShowChipNumber, true);
+    }
 };
```
Comment (DaveOrDead) [inline app/src/js/main.js:19]: We're probably gonna have a few of these type of `for` loops.  I wonder if it's work abstracting to an `addEventToArray` or better named function where you pass in the array, event type and function? e.g 

```
// abstracted to helper function or utils file
function addEvent(array, eventType, func ) {
    for (let i = 0, max= this[ array ]length; i < max; i++) {
        this.[ array ][i].addEventListener(eventType, func, true);
    }
}

// call helper
addEventToArray('click', this.isChippedSelector, this.hideShowChipNumber);
addEventToArray('click', this.updateDateAndLocationText, this.updateDateAndLocationText);

```

Comment (DaveOrDead) [inline app/src/js/main.js:19]: Maybe 'addEventsToNodeList' ? As technically it's not an array

Comment (DaveOrDead) [inline app/src/js/main.js:19]: In fact if we're going fully functional style I guess it should return a copy of the array - no side affects.

File: app/src/js/main.js [hunk 2]
```diff
@@ -29,4 +36,32 @@ petDetectrApp.updateDateAndLocationText = function updateDateAndLocationText(e)
     }
 };
 
+// Toggle the h-hide class on target
+petDetectrApp.toggleDisplay = function toggleDisplayOnOff(target) {
+    /*
+    target.classList.toggle('h-hide');
+    */
+};
+
```
Comment (DaveOrDead) [inline app/src/js/main.js:45]: Is this just a placeholder or a redundant function?

Comment (gary-wood) [inline app/src/js/main.js:45]: It's a generic toggle - wasn't sure if we wanted/needed it

**Pull request closed without merging**